src/integrationTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/AttachExceptionRecordToExistingCaseTest.java

```diff
@@ -432,7 +437,7 @@ public void should_fail_when_exception_record_is_already_attached_to_a_case() th
 
         given()
             .body(attachToCaseRequest(CASE_REF))
-            .post("/callback/{type}", "attach_case")
+            .post("/callback/{type}", ATTACH_CASE)
```

Comment:
could just simply be `.post("/callback/attach_case")` as the name of this whole test class suggests it's about this particular endpoint.

Anyway, not related to this PR if I understand correctly, so could be reverted altogether.

Reply:
yeah even I was thinking that but will revert as it is not part of this PR.